app/javascript/multi-stamp-tubes/components/TubeArraySummary.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <table id="tube_scan_summary" :class="['plate-view', 'pool-colours']">
+    <caption>
+      {{
+        'Summary of tubes scanned into the rack'
+      }}
+    </caption>
+    <thead>
+      <tr>
+        <th class="first-col" />
+        <th id="header_human_barcode" class="headingcell">Human Barcode</th>
+        <th id="header_machine_barcode" class="headingcell">Machine Barcode</th>
+        <th id="header_replicates" class="headingcell">Replicates</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="(row, rowIndex) in tubeSummary" :key="row.barcode">
+        <th class="first-col">
+          {{ (rowIndex + 1).toString() + '.' }}
+        </th>
+        <td :id="`row_human_barcode_index_${rowIndex}`" class="summarycell">
+          {{ row.human_barcode }}
+        </td>
+        <td :id="`row_machine_barcode_index_${rowIndex}`" class="summarycell">
+          {{ row.machine_barcode }}
+        </td>
+        <td :id="`row_replicates_index_${rowIndex}`" class="replicate_cell">
+          {{ row.replicates }}
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</template>
+
+<script>
+export default {
+  name: 'TubeArraySummary',
+  props: {
+    tubes: {
+      type: Array,
+      required: true,
+    },
```

Comment:
You could add a little comment to say a bit more about this `tubes` prop that's passed in. My questions are:
- What are the options for `type`, and why can a tube have an array of types?
- What does it mean for a tube to be required?

Reply:
Added comments

Reply:
Thanks that's useful. I was misunderstanding - the `type` and `required` must refer to the `prop` itself.

Just one more thing - I would change 'related to the position in the tube rack' to 'related to the position in the plate or tube rack' - because it's actually a plate (but could be a tube rack in a future use so may as well leave that in there).